GOVERNANCE.md

```diff
@@ -0,0 +1,150 @@
+Collaborators. All pull requests must be reviewed and accepted by a
+Collaborator with sufficient expertise who is able to take full
+responsibility for the change. In the case of pull requests proposed
+by an existing Collaborator, an additional Collaborator is required
+for sign-off. Consensus should be sought if additional Collaborators
+participate and there is disagreement around a particular
+modification. See _Consensus Seeking Process_ below for further detail
+on the consensus model used for governance.
+
+Collaborators may opt to escalate discussion about significant or
+controversial modifications to the TC as a whole by assigning the
+`express-agenda` tag to a pull request or issue. The TC should serve as the
+final arbiter where required.
+
+For the current list of Collaborators, see the project [README.md][].
+
+### TC Membership
+
+TC seats are not time-limited.  There is no fixed size of the TC.
+However, the expected target is between 6 and 12, to ensure adequate
+coverage of important areas of expertise, balanced with the ability to
+make decisions efficiently.
+
+There is no specific set of requirements or qualifications for TC
+membership beyond these rules.
+
+The TC may add additional members to the TC by consensus. If there are
+any objections to adding any individual member, an attempt should be
+made to resolve those objections following the TC's Consensus Seeking
+Process.
+
+A TC member may be removed from the TC by voluntary resignation, or by
+consensus of all other TC members.
+
+Changes to TC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "TC Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
+TC is not in attendance to participate, then the addition or removal
+is added to the agenda for the subsequent meeting.  This is to ensure
+that all members are given the opportunity to participate in all
+membership decisions.  If a TC member is unable to attend a meeting
+where a planned membership decision is being made, then their consent
+is assumed.
+
+If the TC membership consists of more than 3 active participants, no more than
+1/3 of the TC members may be affiliated with the same employer.  If removal or
+resignation of a TC member, or a change of employment by a TC member, creates a
+situation where more than 1/3 of the TC membership shares an employer, then the
+situation must be immediately remedied by the resignation or removal of one or
+more TC members affiliated with the over-represented employer(s).
+
+### TC Meetings
+
+The TC meets bi-weekly using Google Hangouts on Air or similar tool that allows
+public viewing or participation. A designated moderator approved by the TC runs
+the meeting. A recording of each meeting should be published to either YouTube
+or similar service and all meeting minutes should be posted to the
+expressjs/express GitHub repository.
```

Comment:
What is the purpose of cluttering up the main code repository with meeting minutes? I think it would make sense to have a dedicated repository to hold these things (Express has an entire GitHub organization it can use).